extensions/wikia/AdEngine/js/slot/adUnitBuilder.js

```diff
@@ -0,0 +1,18 @@
+/*global define*/
+define('ext.wikia.adEngine.slot.adUnitBuilder', [
+	'ext.wikia.adEngine.adLogicPageParams',
+], function (page) {
+	'use strict';
+
+	var dfpId = '5441';
+
+	function build(slotName, src) {
+		var pageLevelParams = page.getPageLevelParams();
+		return '/' + dfpId + '/wka.' + pageLevelParams.s0 + '/' + pageLevelParams.s1 +
+			'//' + pageLevelParams.s2 + '/' + src + '/' + slotName;
```

Comment:
What do you think about renaming `pageLevelsParams` to `params` and make:
`return [ params.xxx, params.zzz].join('/')`
Will it be more readable?

Like here:
https://github.com/Wikia/app/blob/dev/extensions/wikia/AdEngine/js/provider/factoryWikiaGpt.js#L70

Another idea: is it worth to reuse adUnitBuilder in factoryWikiaGpt? ;)